gdk_pixbuf2 and gdk3 are pure G-I now, and cairo-gobject does not install
any headers

--- origsrc/gtk3-3.1.1/ext/gtk3/extconf.rb	2017-03-19 17:32:16.717444500 -0500
+++ src/gtk3-3.1.1/ext/gtk3/extconf.rb	2017-03-19 17:32:50.740521900 -0500
@@ -41,11 +41,7 @@ require "mkmf-gnome2"
 depended_packages = [
   "glib2",
   "gobject-introspection",
-  "cairo-gobject",
-  "atk",
   "pango",
-  "gdk_pixbuf2",
-  "gdk3",
 ]
 depended_packages.each do |package|
   directory = "#{package}#{version_suffix}"
